reference/open_pbr_surface.mtlx

```diff
@@ -35,10 +35,12 @@
            doc="Specifies the distance light travels inside the transparent base before it becomes exactly the transmission_color according to Beer's law." />
     <input name="transmission_scatter" type="color3" value="0, 0, 0" uimin="0,0,0" uimax="1,1,1" uiname="Transmission Scatter" uifolder="Transmission" uiadvanced="true"
            doc="Controls the color of light volumetrically scattered inside the transparent base. Suitable for materials with visually significant scattering such as honey, fruit juice, murky water, opalescent glass, or milky glass." />
-    <input name="transmission_scatter_anisotropy" type="float" value="0.0" uimin="0.0" uimax="1.0" uiname="Transmission Anisotropy" uifolder="Transmission" uiadvanced="true"
+    <input name="transmission_scatter_anisotropy" type="float" value="0.0" uimin="-1.0" uimax="1.0" uiname="Transmission Anisotropy" uifolder="Transmission" uiadvanced="true"
            doc="The amount of directional bias, or anisotropy, of the volumetric scattering in the transparent base." />
-    <input name="transmission_dispersion" type="float" value="0.0" uimin="0.0" uisoftmax="100.0" uiname="Transmission Dispersion" uifolder="Transmission" uiadvanced="true"
-           doc="Dispersion amount, describing how much the dielectric index of refraction varies across wavelengths." />
+    <input name="transmission_dispersion_scale" type="float" value="0.0" uimin="0.0" uimax="1.0" uiname="Transmission Dispersion Scale" uifolder="Transmission" uiadvanced="true"
+           doc="Linearly scales the amount of dispersion." />
+    <input name="transmission_dispersion_abbe_number " type="float" value="0.0" uimin="0.0" uisoftmin="9.0" uisoftmax="91.0" uiname="Transmission Dispersion Abbe Number" uifolder="Transmission" uiadvanced="true"
```

Comment:
Just a minor typo, there's a space in the new input name that generates warnings when the shading model is included in MaterialX environments.